src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -50,38 +50,11 @@ static PowerShellProcessInstance()
         /// <param name="workingDirectory">Specifies the initial working directory for the new powershell process.</param>
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-macos`
Basic Job Tests.Basic tests.Can use the user specified working directory parameter with quote
```powershell
Expected exactly '/private/tmp/31d780dd-d55c-488a-8b3c-2e88cfc1e510/My "Dir', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/Job/Jobs.Tests.ps1: line 94
94:             $jobOutput | Should -BeExactly $path.ToString()
```